python/IcebergSinkWithKDS/README.md

```diff
@@ -0,0 +1,249 @@
+2. Critical for environments like Amazon Managed Service for Apache Flink that have strict classloading rules
+3. Always check for new conflicts when updating dependency versions
+
+---
+
+### Requirements
+
+#### Development and build environment requirements
+
+* Python 3.11
+* PyFlink library: `apache-flink==1.20.0`
+* Java JDK 11+ and Maven
+
+> ⚠️ As of 2024-06-27, the Flink Python library 1.19.x may fail installing on Python 3.12.
+> We recommend using Python 3.11 for development, the same Python version used by Amazon Managed Service for Apache Flink
+> runtime 1.20.
+
+> JDK and Maven are used to download and package any required Flink dependencies, e.g. connectors, and
+  to package the application as `.zip` file, for deployment to Amazon Managed Service for Apache Flink.
+
+> This code has been tested and compiled using Java JDK 17.0.15 (Corretto-17.0.15.6.1) and mvn 3.9.9
+
+#### External dependencies
+
+The application expects 1 Kinesis Data Streams.
+
+The stream name is defined in the configuration (see [below](#runtime-configuration)).
+The application defines no default name and region. 
+The [configuration for local development](./application_properties.json) set them, by default, to: 
+
+* `ExampleInputStream`, `us-east-1`
+
+Single-shard Streams in Provisioned mode will be sufficient for the emitted throughput.
+
+#### IAM permissions
+
+The application must have sufficient permissions to publish data to the Kinesis Data Streams.
+
+When running locally, you need active valid AWS credentials that allow publishing data to the Streams.
+
+### Runtime configuration
+
+ When running on Amazon Managed Service for Apache Flink the runtime configuration is read from *Runtime Properties*.
+
+ When running locally, the configuration is read from the [`resources/flink-application-properties-dev.json`](resources/flink-application-properties-dev.json) file located in the resources folder.
+
+Runtime parameters:
+
+| Group ID        | Key                    | Mandatory | Example Value (default for local)    | Notes                         |
+|-----------------|------------------------|-----------|--------------------------------------|-------------------------------|
+| `InputStream0`  | `stream.arn`          | Y         | `arn:aws:kinesis:<region>:<accountId>:stream/ExampleInputStream`                 | Input stream ARN                 |
+| `InputStream0`  | `aws.region`           | Y         | `us-east-1`                          | Region for the input stream.  |
+| `InputStream0`  | `flink.stream.initpos` | N         | `LATEST`                             | Start position in the input stream. `LATEST` by default |
+| `OutputStream0` | `catalog.name`          | Y         | `glue_catalog` | Catalog name to defined               |
+| `OutputStream0` | `aws.region`           | Y         | `us-east-1`                          | Region for the output Iceberg table and catalog.        |
```

Comment:
Please rename the group to `OutputIceberg` or `Iceberg` anything sensible. 
This is no longer a "stream"